src/Products.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Meant to handle special Product features.
+ */
+class Products {
+
+  const POST_TYPE_PRODUCT = 'product';
+
+  const TAX_PRODUCT_CAT = 'product_cat';
+
+  const FIELD_ENFORCE_CAT_LINKS = Plugin::PREFIX . '_enforce_main_category_links';
+
+  /**
+   * Stores the product base link.
+   *
+   * @var string
+   */
+  private string $product_base_link;
+
+  /**
+   * Initialize the product module.
+   */
+  public static function init(): void {
+    $obj = new self();
```

Comment:
This is not how call methods, please have a look everywhere else, and in my following comment.
```suggestion
```